packages/services/f-feature-management-vue3/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@justeat/f-feature-management-vue3",
+  "version": "0.1.0",
+  "description": "Service for querying feature values provided by Feature Management, made reactive for Vue3.",
+  "main": "dist/f-feature-management-vue3.umd.js",
+  "module": "dist/f-feature-management-vue3.es.js",
+  "source": "src/index.js",
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/services/f-feature-management",
+  "contributors": [
+    "Github contributors <https://github.com/justeat/fozzie-components/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeat/fozzie-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/fozzie-components/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn build",
+    "build": "vite build",
+    "lint": "vue-cli-service lint --no-fix",
+    "lint:fix": "yarn lint --fix",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@babel/core": "7.14.8",
+    "@babel/preset-env": "7.14.9",
+    "@vue/cli-plugin-eslint": "3.9.2",
+    "babel-loader": "8.1.0",
+    "eslint-plugin-json-schema-validator": "1.0.48",
+    "eslint-plugin-jsonc": "1.3.1",
+    "jest": "25.4.0",
+    "jest-teamcity": "1.7.0",
+    "vite": "2.4.4",
+    "@vue/reactivity": "3.2.11"
+  },
+  "peerDependencies": {
```

Comment:
I've made these peers in an attempt to keep bundle size down in case multiple components take on a dependency on this package.  I _think_ this makes sense?

Reply:
I definitely agree that `f-feature-management` should be a peer dependency, but I'm not sure if having the reactivity package as one too is worthwhile.
Also, is it a mistake to have it in `devDependencies` too?

Reply:
I only made it a `devDependency` because the tests require it (and didn't work till I added it).  It is used at runtime so it can't only be a `devDependency`.

Reply:
I think `devDependencies` is correct for an npm package as it'll be included in the final bundle rather than used in the consuming application. 

Putting it in `dependencies` means that when you run `yarn install` in a consuming app, the dep will be downloaded but not necessarily used.

Reply:
> I only made it a `devDependency` because the tests require it (and didn't work till I added it). It is used at runtime so it can't only be a `devDependency`.

Yeah, `peerDependencies` should go in `devDependencies` and then excluded in the build config so that we can run the code in isolation or in unit tests.